cmd/operator-sdk/alpha/bundle/cmd.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bundle
+
+import (
+	"github.com/spf13/cobra"
+)
+
+type bundleCmd struct {
+	directory      string
+	imageTag       string
+	imageBuilder   string
+	packageName    string
+	channels       []string
+	channelDefault string
+	overwrite      bool
+	generateOnly   bool
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bundle",
+		Short: "Work with operator bundle metadata and bundle images",
```

Comment:
Could we add here just the info where it is used? Such as: WDYT?
```suggestion
		Short: "Generate operator bundle metadata and bundle images to work with OLM ",
```